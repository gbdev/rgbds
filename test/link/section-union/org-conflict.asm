IF !DEF(SECOND)
	def ADDR = $BEEF
ELSE
	def ADDR = $BABE
ENDC

SECTION UNION "conflicting address", SRAM[ADDR]
	db
