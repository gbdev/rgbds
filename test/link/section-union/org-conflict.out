error: Section "conflicting address" is defined with conflicting addresses $beef and $babe
---
error: <stdin>(16):
    Section already declared as fixed at different address $beef
FATAL: <stdin>(16):
    Cannot create section "conflicting address" (1 error)
